src/helper/markdown/lineQueries.ts

```diff
@@ -0,0 +1,63 @@
+// Special Tokens
+enum MarkdownTokens {
+    H1 = '# ',
+    H2 = '## ',
+    H3 = '### ',
+    H4 = '#### ',
+    H5 = '##### ',
+    H6 = '###### ',
+}
+
+function isHeadingLevel1(line: string): boolean {
+    return line.startsWith(MarkdownTokens.H1);
+}
+
+function isHeadingLevel2(line: string): boolean {
+    return line.startsWith(MarkdownTokens.H2);
+}
+
+function isHeadingLevel3(line: string): boolean {
+    return line.startsWith(MarkdownTokens.H3);
+}
+
+function isHeadingLevel4(line: string): boolean {
+    return line.startsWith(MarkdownTokens.H4);
+}
+
+function isHeadingLevel5(line: string): boolean {
+    return line.startsWith(MarkdownTokens.H5);
+}
+
+function isHeadingLevel6(line: string): boolean {
+    return line.startsWith(MarkdownTokens.H6);
+}
+
+// Line formatter
+export function formatHtmlForMarkdownLine(line: string) {
+    let formattedLine = line;
+
+    if (isHeadingLevel1(line)) {
```

Comment:
can you chain `.replace` with regex instead of using `if/else`?

Reply:
Hi, can you please elaborate on how you want it done. Maybe with some example.

Reply:
something like this

```ts
let formattedLine = line
      .replace(regex for h1, '<h1>$1</h1>') // for h1
      .replace(regex for h1, '<h2>$1</h2>') // for h2
      ..... 
return formattedLine;
```

Reply:
@avelynhc Please review the latest commit